go.mod

```diff
@@ -1,4 +1,6 @@
-go 1.21
+go 1.22
```

Comment:
Found the culprit: https://github.com/cosmos/cosmos-db/blob/main/go.mod#L3.
Are we using any go 1.22 features in cosmos-db?

Reply:
We are not to me knowledge.

Reply:
Okay, then maybe we should stay in go 1.19 there and just use a go matrix for testing from 1.19 to 1.22 instead of just 1.22